Jenkinsfile_CNP

```diff
@@ -6,4 +6,33 @@ def type = "java"
 def product = "pcs"
 def component = "api"
 
-withPipeline(type, product, component) {}
+withPipeline(type, product, component) {
+  onPR() {
+    afterAlways('functionalTest:preview') {
```

Comment:
afterAlways runs evern when functional tests fail, i dont think its worth running these when functional tests itself fail. Also, may be worth doing before functional tests as this is more of a smoke functional test?